File: katas/8 kyu/sentence-smash/solution.php
<!-- sentence-smash -->
<?php
use PHPUnit\Framework\TestCase;
/* ------------------------------- MY SOLUTION ------------------------------ */
function smash(array $words): string {
  return implode(" " , $words);
}
/* ----------------------------- CLEVER SOLUTION ---------------------------- */
/* ---------------------------------- TEST ---------------------------------- */
class SmashTest extends TestCase {
  public function testExamples() {
    $this->assertSame('', smash([]));
    $this->assertSame('hello', smash(["hello"]));
    $this->assertSame('hello world', smash(["hello", "world"]));
  }
}

File: katas/8 kyu/sentence-smash/solution.py
# sentence-smash
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
def smash(words):
  return " ".join(words)
# ------------------------------ CLEVER SOLUTION ----------------------------- #
# ----------------------------------- TEST ----------------------------------- #
test.assert_equals(smash([]), "")
test.assert_equals(smash(["hello"]), "hello")
test.assert_equals(smash(["hello", "world"]), "hello world")
test.assert_equals(smash(["hello", "amazing", "world"]), "hello amazing world")
test.assert_equals(smash(["this", "is", "a", "really", "long", "sentence"]), "this is a really long sentence")